core/routes.go

```diff
@@ -26,65 +28,206 @@ func updateRoutes(pg *postgres) {
 
 	existing, new := checkRouteExists(pg, aircrafts)
 
-	if len(new) > 50 {
-		new = new[:50]
```

Comment:
the API doesn't enforce a hard limit (yet), however, the existing consumers all request up to 100 routes at a time. I don't think that requesting more than that would break things, but in an abundance of caution (and to reduce spikes on restart)...

Reply:
Agree - wiedehopf mentioned similar - added a cap of 100 routes per call. Any overflow should just get picked up in the next iteration.